utils/flist/techmap_vlog.flist

```diff
@@ -1,123 +0,0 @@
-techmap/inferred/SyncSpRamBeNx64.sv
-techmap/virtex7/sram_b/unisim_sram_b_10abits.v
-techmap/virtex7/sram_b/unisim_sram_b_16abits.v
-techmap/virtex7/sram_b/unisim_sram_b_14abits.v
-techmap/virtex7/sram_b/unisim_sram_b_20abits.v
-techmap/virtex7/sram_b/unisim_sram_b_7abits.v
-techmap/virtex7/sram_b/unisim_sram_b_18abits.v
-techmap/virtex7/sram_b/unisim_sram_b_11abits.v
-techmap/virtex7/sram_b/unisim_sram_b_15abits.v
-techmap/virtex7/sram_b/unisim_sram_b_6abits.v
-techmap/virtex7/sram_b/unisim_sram_b_19abits.v
-techmap/virtex7/sram_b/unisim_sram_b_12abits.v
-techmap/virtex7/sram_b/unisim_sram_b_17abits.v
-techmap/virtex7/sram_b/unisim_sram_b_13abits.v
-techmap/virtex7/sram_b/unisim_sram_b_9abits.v
-techmap/virtex7/sram_b/unisim_sram_b_8abits.v
-techmap/virtex7/sgmii_vcu118_reset_sync.v
-techmap/virtex7/mem/BRAM_1024x16.v
-techmap/virtex7/mem/BRAM_16384x1.v
-techmap/virtex7/mem/BRAM_2048x8.v
-techmap/virtex7/mem/BRAM_4096x4.v
-techmap/virtex7/mem/BRAM_512x32.v
-techmap/virtex7/mem/BRAM_8192x2.v
-techmap/virtexu/sram_b/unisim_sram_b_10abits.v
-techmap/virtexu/sram_b/unisim_sram_b_16abits.v
-techmap/virtexu/sram_b/unisim_sram_b_14abits.v
-techmap/virtexu/sram_b/unisim_sram_b_20abits.v
-techmap/virtexu/sram_b/unisim_sram_b_7abits.v
-techmap/virtexu/sram_b/unisim_sram_b_18abits.v
-techmap/virtexu/sram_b/unisim_sram_b_11abits.v
-techmap/virtexu/sram_b/unisim_sram_b_15abits.v
-techmap/virtexu/sram_b/unisim_sram_b_6abits.v
-techmap/virtexu/sram_b/unisim_sram_b_19abits.v
-techmap/virtexu/sram_b/unisim_sram_b_12abits.v
-techmap/virtexu/sram_b/unisim_sram_b_17abits.v
-techmap/virtexu/sram_b/unisim_sram_b_13abits.v
-techmap/virtexu/sram_b/unisim_sram_b_9abits.v
-techmap/virtexu/sram_b/unisim_sram_b_8abits.v
-techmap/virtexu/sgmii_vcu118_reset_sync.v
-techmap/virtexu/mem/BRAM_1024x16.v
-techmap/virtexu/mem/BRAM_16384x1.v
-techmap/virtexu/mem/BRAM_2048x8.v
-techmap/virtexu/mem/BRAM_4096x4.v
-techmap/virtexu/mem/BRAM_512x32.v
-techmap/virtexu/mem/BRAM_8192x2.v
-techmap/virtexup/sram_b/unisim_sram_b_10abits.v
-techmap/virtexup/sram_b/unisim_sram_b_16abits.v
-techmap/virtexup/sram_b/unisim_sram_b_14abits.v
-techmap/virtexup/sram_b/unisim_sram_b_20abits.v
-techmap/virtexup/sram_b/unisim_sram_b_7abits.v
-techmap/virtexup/sram_b/unisim_sram_b_18abits.v
-techmap/virtexup/sram_b/unisim_sram_b_11abits.v
-techmap/virtexup/sram_b/unisim_sram_b_15abits.v
-techmap/virtexup/sram_b/unisim_sram_b_6abits.v
-techmap/virtexup/sram_b/unisim_sram_b_19abits.v
-techmap/virtexup/sram_b/unisim_sram_b_12abits.v
-techmap/virtexup/sram_b/unisim_sram_b_17abits.v
-techmap/virtexup/sram_b/unisim_sram_b_13abits.v
-techmap/virtexup/sram_b/unisim_sram_b_9abits.v
-techmap/virtexup/sram_b/unisim_sram_b_8abits.v
-techmap/virtexup/sgmii_vcu118_reset_sync.v
-techmap/virtexup/mem/BRAM_1024x16.v
-techmap/virtexup/mem/BRAM_16384x1.v
-techmap/virtexup/mem/BRAM_2048x8.v
-techmap/virtexup/mem/BRAM_4096x4.v
-techmap/virtexup/mem/BRAM_512x32.v
-techmap/virtexup/mem/BRAM_8192x2.v
```

Comment:
why deleting FPGA files?

Reply:
I am not deleting them. techmap_v*.flist is now automatically generated once "check_all_srcs" target inside rtl.mk is called. This is required due to different memory wrapper names according to the user size preference.